### ISSUE TITLE
Override of program_icon.ico no longer works in v1.0.3

### ISSUE THREAD

**Issue by jamwil** [chriskiehl/Gooey#498]
- OS: Windows 10
- Python Version: 3.7.5 64-bit
- Gooey Version: 1.0.3
- Description: `program_icon.ico` was previously overridden by specifying `image_dir` in the initialization decorator. While image_dir continues to work for the other images that can be overridden, my program icon located in the Title Bar on Windows 10 is back to Gooey's new default icon.
```python
@Gooey(
    program_name="Program Name",
    image_dir="static",
    default_size=(1050, 550),
    progress_regex=r"^.* (\d+)%$",
    disable_stop_button=True
)
def main():
    ...
```
Thanks!

**Comment by dusans** [comment]
It has to be a .png file since 1.0.3.

**Comment by codygulley** [comment]
This needs to be updated in the docs. 
[Readme](https://github.com/chriskiehl/Gooey#customizing-icons)  still has it listed as program_icon.ico

**Comment by chriskiehl** [comment]
Ugh. Bad job, me :(

Will fix post vacation!

**Comment by chriskiehl** [comment]
[Fixed on the release branch](5e4959b7daad5912db3c37eed9627a12c861e244)
